packages/merkle-distributor/src/merkleTree.js

```diff
@@ -0,0 +1,131 @@
+const web3 = require("web3");
+const ethereumjsUtil = require("ethereumjs-util");
+
+class MerkleTree {
+  constructor(elements) {
+    // Filter empty strings and hash elements
+    // NOTE do not double has leafs
+    // this.elements = elements.filter(el => el).map(el => keccakFromString(el));
+    this.elements = elements;
+    // Sort elements
+    this.elements.sort();
+    // Deduplicate elements
+    // this.elements = this.bufDedup(this.elements);
```

Comment:
Commented code?

Reply:
ye does not matter in this context and came from the complied implementation from piedao.